### PR TITLE
Java troubleshooting

### PR DESCRIPTION
**Pull request by drodriguezhdez**

### CODE REVIEW

File: docs/java-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+id: java-troubleshooting
+title: Scope Java Agent troubleshooting
+sidebar_label: Troubleshooting
+---
+
+## Problem
```
Comment (fermayo) [inline docs/java-troubleshooting.md]: Having `Problem` as the title makes the right hand side index useless:
Screenshot 2020-02-15 at 17 25 32

Instead, I'd have the problem as the title, and the solution as the body, something like:

```md
### I don't see my tests in Scope after installing the Scope Java agent

If you don't see any results in Scope after following the [Scope Java agent installation instructions](java-installation.md), check the following:

* **Set a valid `SCOPE_DSN` environment variable.** The Scope Java agent...
```

File: docs/java-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+You do not see data in Scope after [installing the Scope Java agent](java-installation.md).
+
+## Solution
+
+### Have you configured a valid `SCOPE_DSN`?
+
+The Scope Java agent needs a valid `SCOPE_DSN` configured as environment variable.
+
+Keep in mind that `SCOPE_DSN` is different per **namespace** so if you configure a `SCOPE_DSN` from other namespace, although it was from the correct environment, you will not see results in Scope for that repository.
```
Comment (fermayo) [inline docs/java-troubleshooting.md]: `although it was from the correct environment`

What do you mean?

File: docs/java-troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+You can find further information in [Scope Java agent - Environment variables](java-installation.md#environment-variables) section.
+
+### Have you forwarded needed environment variables?
```
Comment (fermayo) [inline docs/java-troubleshooting.md]:
```suggestion
### Have you forwarded the required environment variables?
```

File: docs/java-troubleshooting.md (hunk 2 above)
Comment (drodriguezhdez) [reply docs/java-troubleshooting.md]: I tried to express the fact about "if you are in the correct Scope instance", like SaaS or your on prem environment, it is not sufficient.

Comment (fermayo) [reply docs/java-troubleshooting.md]: I think the likelihood of someone using different Scope instances at the same time is very low, so we can just remove that and should be fine. Most people will only use one Scope instance, either the SaaS one or their on-prem one.

**Comment by drodriguezhdez** [comment]
@fermayo take a look when you have time

File: docs/java-troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+id: java-troubleshooting
+title: Scope Java Agent troubleshooting
+sidebar_label: Troubleshooting
+---
+
+## I don't see my tests in Scope after installing the Scope Java agent
+
+If you don't see any results in Scope after following the [Scope Java agent installation](java-installation.md), check the following:
+
+#### **Have you configured a valid `SCOPE_DSN`?**
```
Comment (fermayo) [inline docs/java-troubleshooting.md]:
```suggestion
#### Have you configured a valid `SCOPE_DSN`?
```

File: docs/java-troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+The Scope Java agent needs a valid `SCOPE_DSN` configured as environment variable.
+
+Keep in mind that `SCOPE_DSN` is different per **namespace** so if you configure a `SCOPE_DSN` from other namespace, you will not see results in Scope for that repository.
+
+You can find further information in [Scope Java agent - Environment variables](java-installation.md#environment-variables) section.
+
+#### **Have you forwarded the required environment variables?**
```
Comment (fermayo) [inline docs/java-troubleshooting.md]:
```suggestion
#### Have you forwarded the required environment variables?
```

File: docs/java-troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+If you are executing your tests in a **container**, you need to forward several environment variables depending on your CI provider, so the agent can autodetect the build information.
+
+You can find further information in [Scope Java agent - Running tests inside a container](java-installation.md#running-tests-inside-a-container) section.
+
+#### **Have you checked the compatibility of Scope Java agent with your project?**
```
Comment (fermayo) [inline docs/java-troubleshooting.md]:
```suggestion
#### Have you checked the compatibility of Scope Java agent with your project?
```

File: docs/java-troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+The Scope Java agent needs a valid `SCOPE_DSN` configured as environment variable.
+
+Keep in mind that `SCOPE_DSN` is different per **namespace** so if you configure a `SCOPE_DSN` from other namespace, you will not see results in Scope for that repository.
+
+You can find further information in [Scope Java agent - Environment variables](java-installation.md#environment-variables) section.
+
+#### **Have you forwarded the required environment variables?**
+
+If you are executing your tests in a **container**, you need to forward several environment variables depending on your CI provider, so the agent can autodetect the build information.
```
Comment (fermayo) [inline docs/java-troubleshooting.md]:
```suggestion
If you are executing your tests in a container, you need to forward several environment variables depending on your CI provider, so the agent can autodetect the build information.
```

**Pull request merged**